Extension/src/LanguageServer/settings.ts

```diff
@@ -202,15 +202,15 @@ export class CppSettings extends Settings {
     public get clangFormatStyle(): string | undefined { return super.Section.get<string>("clang_format_style"); }
     public get clangFormatFallbackStyle(): string | undefined { return super.Section.get<string>("clang_format_fallbackStyle"); }
     public get clangFormatSortIncludes(): string | undefined { return super.Section.get<string>("clang_format_sortIncludes"); }
-    public get experimentalFeatures(): string | undefined { return super.Section.get<string>("experimentalFeatures"); }
+    public get experimentalFeatures(): boolean | undefined { return super.Section.get<string>("experimentalFeatures")?.toLowerCase() === "enabled"; }
     public get suggestSnippets(): boolean | undefined { return super.Section.get<boolean>("suggestSnippets"); }
-    public get intelliSenseEngine(): string | undefined { return super.Section.get<string>("intelliSenseEngine"); }
-    public get intelliSenseEngineFallback(): string | undefined { return super.Section.get<string>("intelliSenseEngineFallback"); }
+    public get intelliSenseEngine(): boolean | undefined { return super.Section.get<string>("intelliSenseEngine")?.toLowerCase() === "enabled"; }
```

Comment:
It turns out I need to roll back the changes made to intellisenseEngine because the options are not only enabled,disabled. I don't know why I missed this the first time

Reply:
While `intelliSenseEngine` appears to have been incorrectly converted to a bool, IMHO consistency is important.   Currently, the settings getters are consistently named to match the setting (with some inconsistency).  Whatever we do, we should intend to be consistent, or to make things more (not less) consistent.  There are lots of (all) existing bool settings that do not use the 'is' syntax.  If we want to use the "is" syntax consistently, we should apply it to the rest of code base as well.  IMHO, it seems that the other settings, with only "Enabled" and "Disabled" values, actually read intuitively as booleans.  I think it would be better to keep the setting names consistent.

Reply:
I don't recall asking for 'is' at the beginning of these. Boolean naming is typically a predicate of some kind, never a noun. 'Is' is an acceptable way to start a predicate, but is not required. Many verbs work and there are several examples here.

With the exception of the `vcFormat.space` and `vcFormat.newLine` options where "insert" is implied to keep the name shorter, there are only a few boolean `get` properties here that don't follow that convention. The settings with enum values were named appropriately based on what they were.  Had they been defined as booleans in package.json, they would have been given predicate names. If you want to change the type to boolean behind the scenes, that's fine, but it is best to fix the name too so that the code reads well.

This is my recommendation. You are welcome to accept or reject. I will not block the PR for this.